pkg/controller/template/template_controller.go

```diff
@@ -117,16 +123,24 @@ func New(
 		DeleteFunc: ctrl.deleteFeature,
 	})
 
+	storageInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		//AddFunc:    ctrl.addFeature,
+		//UpdateFunc: ctrl.updateFeature,
+		//DeleteFunc: ctrl.deleteFeature,
+	})
```

Comment:
i'm still working on these

Reply:
Won't this cause problems if for example `addFeature` is called and provided a `Storage` object which `addFeature` casts to a `FeatureGate`?
https://github.com/openshift/machine-config-operator/blob/f08e554770af2c879ea32a68c3d66c068ee0da83/pkg/controller/template/template_controller.go#L219-L223